.all-contributorsrc

```diff
@@ -60,6 +60,7 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/9108586?v=4",
       "profile": "https://czaplicki.it/",
       "contributions": [
+      	"test",
```

Comment:
I think this should be done via the github ui to avoid issues? 

you yourself can call it, `@all-contributors add @mdczaplicki for test and code` or something